pilot/pkg/model/gateway.go

```diff
@@ -189,10 +189,14 @@ func MergeGateways(gateways []gatewayWithInstances, proxy *Proxy, ps *PushContex
 					// Envoy will reject config that has multiple filter chain matches with the same matching rules.
 					// To avoid this, we need to make sure we don't have duplicated hosts, which will become
 					// SNI filter chain matches.
+
```

Comment:
Please consider changing it like this			
	// When there is Bind specified in the Gateway, the listener is built per IP instead of
					// sharing one wildcard listener. So different Gateways can
					// have same host as long as they have different Bind.

Reply:
Done